flank-scripts/src/main/kotlin/flank/scripts/github/objects/GitHubUpdateIssue.kt

```diff
@@ -0,0 +1,20 @@
+package flank.scripts.github.objects
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class GitHubUpdateIssueRequest(
+    val title: String = "",
+    val body: String = "",
+    val state: IssueState = IssueState.OPEN,
+    val milestone: Int? = null,
+    val labels: List<String> = emptyList(),
+    val assignees: List<String> = emptyList()
+)
+
+@Serializable
+enum class IssueState {
+    @SerialName("open") OPEN,
```

Comment:
I want to confirm if the kotlinx serialization is fully compatible with the GSON we are making use of
Else if might result in us changing all the annotations.

Reply:
I think they are, not 100% sure though. In `flank-scripts` module we do not use GSON.

Reply:
I saw here we are DE serializing with GSON so just want to confirm.

Reply:
Wait what? We should not do that

Reply:
@Sloox could you give an example? I am unable to find